scripts/nbconverted/explore_offset.py

```diff
@@ -0,0 +1,259 @@
+
+
+# In[16]:
+
+
+# Get gene ids with the highest negative weight from the max feature selected
+percentile = 5
+threshold = np.percentile(genes_max_feature, percentile)
+print("Threshold cutoff is {}".format(threshold))
+lowest_genes = genes_max_feature[genes_max_feature < threshold].index
+
+
+# In[17]:
+
+
+# Compare the overlap of genes in set A and highest positive weighted genes in the max feature
+venn2([set(highest_genes), geneSetA_set], set_labels = ('High positive weight genes in feature {}'.format(max_feature), 'Group A genes'))
+plt.show()
+
+
+# In[18]:
+
+
+# Compare the overlap of genes in set B and highest positive weighted genes in the max feature
+venn2([set(highest_genes), geneSetB_set], set_labels = ('High positive weight genes in feature {}'.format(max_feature), 'Group B genes'))
+plt.show()
+
+
+# In[28]:
+
+
+# Compare the overlap of genes in set A and highest negative weighted genes in the max feature
+venn2([set(lowest_genes), geneSetA_set], set_labels = ('High negative weight genes in feature {}'.format(max_feature), 'Group A genes'))
+plt.show()
+
+
+# In[29]:
+
+
+# Compare the overlap of genes in set B and highest negative weighted genes in the max feature
+venn2([set(lowest_genes), geneSetB_set], set_labels = ('High negative weight genes in feature {}'.format(max_feature), 'Group B genes'))
+plt.show()
+
+
+# In[21]:
+
+
+# Get gene weights for min latent feature
+genes_min_feature = weight[int(min_feature)]
+sns.distplot(genes_min_feature)
+
+
+# In[22]:
+
+
+# Get gene ids with the highest positive weight from the min feature selected
+percentile = 95
+threshold = np.percentile(genes_min_feature, percentile)
+print("Threshold cutoff is {}".format(threshold))
+highest_genes = genes_min_feature[genes_min_feature > threshold].index
+
+
+# In[23]:
+
+
+# Get gene ids with the highest negative weight from the min feature selected
+percentile = 5
+threshold = np.percentile(genes_min_feature, percentile)
+print("Threshold cutoff is {}".format(threshold))
+lowest_genes = genes_min_feature[genes_min_feature < threshold].index
+
+
+# In[33]:
+
+
+# Compare the overlap of genes in set A and highest positive weighted genes in the min feature
+venn2([set(highest_genes), geneSetA_set], set_labels = ('High negative weight genes in feature {}'.format(min_feature), 'Group A genes'))
+plt.show()
+
+
+# In[32]:
+
+
+# Compare the overlap of genes in set B and highest positive weighted genes in the min feature
+venn2([set(highest_genes), geneSetB_set], set_labels = ('High negative weight genes in feature {}'.format(min_feature), 'Group B genes'))
+plt.show()
+
+
+# In[30]:
+
+
+# Compare the overlap of genes in set A and highest negative weighted genes in the min feature
+venn2([set(lowest_genes), geneSetA_set], set_labels = ('High negative weight genes in feature {}'.format(min_feature), 'Group A genes'))
+plt.show()
+
+
+# In[31]:
+
+
+# Compare the overlap of genes in set B and highest negative weighted genes in the min feature
```

Comment:
The min and max features have the same overlap? This is suspicious to me

Reply:
I noticed this as well, but the analysis was a bit rushed.  Will need to look back into it in the next PR

Reply:
#11 ?

Reply:
Issue https://github.com/greenelab/Pseudomonas_latent_spaces/issues/13